model/delivery/AutoStartProctoredDeliveryService.php

```diff
@@ -91,9 +91,12 @@ private function changeStateToAuthorized(DeliveryExecution $deliveryExecution):
         $deliveryExecution->getImplementation()->setState(ProctoredDeliveryExecution::STATE_AUTHORIZED);
 
         $this->logger->info(
-            get_called_class() . ' changes the status of delivery execution to authorized',
+            sprintf(
+                'Changes in the status of delivery execution to authorized. Delivery Execution Identifier: %s',
```

Comment:
I fail to understand the message here.
Did you mean something like that?
```suggestion
                '"%s" Delivery Execution's status changed to authorized',
```